src/engine/DnCManager.cpp

```diff
@@ -347,39 +329,23 @@ void DnCManager::printResult()
     {
         std::cout << "sat\n" << std::endl;
 
-        extractSolution( *_baseInputQuery );
-
-        Vector<double> inputVector( _baseInputQuery->getNumInputVariables() );
```

Comment:
There is a danger that the network-level reasoner's output variable is different from the actual output variable. Also, the output assignment might be inconsistent with the satisfying assignment returned by the API call. Disabling this re-computation with NLR for now and left a TODO.